README.md

```diff
@@ -176,75 +114,12 @@ secretproviderclasses.secrets-store.csi.x-k8s.io
-
-    # [REQUIRED FOR VAULT PROVIDER]
-    kubectl apply -f https://raw.githubusercontent.com/hashicorp/secrets-store-csi-driver-provider-vault/master/deployment/provider-vault-installer.yaml --namespace $NAMESPACE
-
-    ```
-
-    You should see the following pods deployed for the provider(s) you selected. For example, for the Azure Key Vault provider:
-
-    ```bash
-    csi-secrets-store-provider-azure-pksfd             2/2     Running   0          4m
-    csi-secrets-store-provider-azure-sxht2             2/2     Running   0          4m
-    ```
-
-1. Create a `secretproviderclasses` resource to provide provider-specific parameters for the Secrets Store CSI driver. Follow [specific deployment steps](#providers) for the selected provider to update all required fields [see example secretproviderclass](pkg/providers/azure/examples/v1alpha1_secretproviderclass.yaml).
-
-      ```yaml
-      apiVersion: secrets-store.csi.x-k8s.io/v1alpha1
-      kind: SecretProviderClass
-      metadata:
-        name: azure-kvname
-      spec:
-        provider: azure                   # accepted provider options: azure or vault
-        parameters:
-          usePodIdentity: "false"         # [OPTIONAL for Azure] if not provided, will default to "false"
-          keyvaultName: "kvname"          # the name of the KeyVault
-          objects:  |
-            array:
-              - |
-                objectName: secret1
-                objectType: secret        # object types: secret, key or cert
-                objectVersion: ""         # [OPTIONAL] object versions, default to latest if empty
-              - |
-                objectName: key1
-                objectType: key
-                objectVersion: ""
-          resourceGroup: "rg1"            # the resource group of the KeyVault
-          subscriptionId: "subid"         # the subscription ID of the KeyVault
-          tenantId: "tid"                 # the tenant ID of the KeyVault
-
-      ```
-1. Update your [deployment yaml](pkg/providers/azure/examples/nginx-pod-secrets-store-inline-volume-secretproviderclass.yaml) to use the Secrets Store CSI driver and reference the `secretProviderClass` resource created in the previous step
-
-    ```yaml
-    volumes:
-      - name: secrets-store-inline
-        csi:
-          driver: secrets-store.csi.k8s.io
-          readOnly: true
-          volumeAttributes:
-            secretProviderClass: "azure-kvname"
-    ```
-
-1. Deploy your resource with the inline CSI volume using the Secrets Store CSI driver
-
-    ```bash
-    kubectl apply -f pkg/providers/azure/examples/nginx-pod-secrets-store-inline-volume-secretproviderclass.yaml
-    ```
-
-1. Validate the pod has access to the secret from your secrets store instance:
-
-    ```bash
-    kubectl exec -it nginx-secrets-store-inline ls /mnt/secrets-store/
-    secret1
-    ```
+Select a provider from the [list of supported providers](#providers) and deploy the provider yaml
+
```

Comment:
Instead of pointing to the `Providers` section, lets point to the projects directly. How about?
```
Select a provider from the following list, then follow the installation steps for that provider:
-  [Azure Provider](https://github.com/Azure/secrets-store-csi-driver-provider-azure#install-the-azure-key-vault-provider)
-  [Vault Provider](https://github.com/hashicorp/secrets-store-csi-driver-provider-vault)
```

Reply:
Updated, PTAL!